packages/cli/src/commands/lint.ts

```diff
@@ -36,9 +36,21 @@ const lintCommand: CommandModule = {
         },
       })
       .middleware((argv: Dictionary<unknown>) => {
-        const [format] = argv.format as string[] & { 0: string };
-        if (typeof argv.output === 'string') {
-          argv.output = { [format]: argv.output };
+        const formats = argv.format as string[] & { 0: string };
+        if (argv.output === void 0) {
+          argv.output = { [formats[0]]: '<stdout>' };
+        } else if (typeof argv.output === 'string') {
+          argv.output = { [formats[0]]: argv.output };
+        } else {
+          const output = argv.output as Dictionary<unknown>;
+          if (Object.keys(output).length + 1 >= formats.length) {
+            return;
```

Comment:
Should we maybe report an error in this case?

Reply:
This should be caught by https://github.com/stoplightio/spectral/blob/fdcdabd99e8cf1a87935ea4cd02722ac36e219e1/packages/cli/src/commands/lint.ts#L77